scylla/src/frame/response/result.rs

```diff
@@ -341,10 +341,16 @@ pub struct ResultMetadata {
     pub col_specs: Vec<ColumnSpec>,
 }
 
+#[derive(Debug, Copy, Clone)]
+pub struct PKIndex {
```

Comment:
let's be more explicit and name this one PrimaryKeyIndex. I know that later "pk" is used in variable names, but at least people would be more likely to correctly decipher the acronym :)

Reply:
Isn't PK the acronym of PartitionKey here? I admit I was not sure between partition key and primary key, that's why I've left PK.